workspaces/tech-insights/.changeset/chilly-badgers-thank.md

```diff
@@ -0,0 +1,8 @@
+---
+'@backstage-community/plugin-tech-insights-backend': patch
+---
+
+Add auth token to custom fact retriever example
```

Comment:
We can simplify this by removing this line 👍

Reply:
Done. Thanks. 👍